core/src/test/java/io/opentelemetry/android/export/BufferDelegatingSpanExporter.kt

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.android.export
+
+import io.mockk.mockk
+import io.mockk.spyk
+import io.mockk.verify
+import io.opentelemetry.sdk.testing.assertj.OpenTelemetryAssertions.assertThat
+import io.opentelemetry.sdk.testing.exporter.InMemorySpanExporter
+import io.opentelemetry.sdk.trace.data.SpanData
+import org.junit.Test
+
+class BufferDelegatingSpanExporterTest {
+    @Test
+    fun `test setDelegate`() {
+        val bufferDelegatingSpanExporter = BufferDelegatingSpanExporter()
+        val spanExporter = InMemorySpanExporter.create()
+
+        val spanData: SpanData = mockk<SpanData>()
```

Comment:
No need to specify generic type here
```suggestion
        val spanData: SpanData = mockk()
```

Reply:
done